DBZ-5878: Update the Debezium UI Documentation page to include the latest additional steps and UI screenshots (#4140)
debezium/debezium: documentation/modules/ROOT/pages/operations/debezium-ui.adoc

The main page of the UI displays all the registered connectors.  Some of the highlights of the main page are as follows:

* *Kafka connect cluster* can be selected via the dropdown in the header.
* Connector table shows each connector with it's type (MongoDB, MySQL, Oracle, PostgreSQL and SQL Server), connector status and connector tasks.
* A connector row can be expanded to show more details, as shown below with the 'testPostgres' connector.  Metrics are shown in the expansion area (*Note:* this feature is still under development and not functional yet).  Connector tasks are shown, with ability to *Restart* the task if desired.

> * The kebab menu at the right of each connector row provides actions which allow the user to *Pause, Resume, Restart or Delete* the connector.

Author: updated

The basic connection properties for the selected connector are entered in step 2, and the properties must be validated before proceeding.  Advanced connection properties are also provided in a separate section of this step.  Upon successful validation, the user may proceed to the next steps (Additional properties) - or they can elect to bypass the additional properties and proceed directly to Review by clicking on to *Review and finish* button in the wizard footer.

image::debezium-ui-step2.png[{prodname} UI Create connector step2]

==== Create Connector - Additional properties (optional)
The Additional properties are optional and can be summarized as follows:

* *Filter definition* - entry of *regular expressions* which define the filters for inclusion/exclusion of the items that will be included for CDC.  The included items are displayed as the filters are entered and applied.
* *Transformations* - enable single message at a time modification. This can use to to modify records before they are sent to Apache Kafka by applying them to the {prodname} connectors. Read more about the transformation here xref:{link-transformations}[transformations]

> * *Topic creation* - define automatic topic creation default properties and groups.

Reviewer: ```suggestion
* *Topic creation* - Define default properties and groups for enabling automatic topic creation.
```

* *Data options* - *Snapshot* and *Mapping* properties (optional).  The defaults can be viewed and changed if desired.
* *Runtime options* - *Engine* and *Heartbeat* properties (optional).  The defaults can be viewed and changed if desired.

> * *Custom properties* - enable pass-thru of additional properties that are not covered by the previous sections can be entered here e.g database.* and database.history.* options. The user can enter any valid property name and value.

Reviewer: ```suggestion
* *Custom properties* - Enable pass-through of additional properties that are not specified in the previous sections, such as `database.*` and `database.history.*` options. You can enter any valid property name and value.
```
